package/CHANGELOG

```diff
@@ -14,6 +14,17 @@ The rules for this file:
 
 
 -------------------------------------------------------------------------------
+03/26/24 orbeckst, SampurnaM
+
+ * 2.8.0
+
+Fixes:
+ * Added a tqdm progress bar for ``MDAnalysis.analysis.pca.PCA.transform()``
+    It can be displayed or disabled using ``verbose`` parameter 
+    ``True = display | False = display off``
+     (PR #4531)
+
+
```

Comment:
Add everything to the existing 2.8.0 block: It should look like
```

??/??/?? IAlibay, HeetVekariya, marinegor, lilyminium, RMeli,
         ljwoods2, aditya292002, pstaerk, PicoCentauri, BFedder.
         SampurnaM

 * 2.8.0

```

and then add under **Enhancements**
```
* Added a tqdm progress bar for `MDAnalysis.analysis.pca.PCA.transform()`
  (PR #4531)
```
(Minimal detail is sufficient, and break lines a 79 characters)